File: src/main/java/com/lykke/matching/engine/daos/Candle.java
package com.lykke.matching.engine.daos;

import com.microsoft.azure.storage.table.TableServiceEntity;

public class Candle extends TableServiceEntity {
    String data;

    public Candle() {
    }

    public Candle(String partitionKey, String rowKey, String data) {
        super(partitionKey, rowKey);
        this.data = data;
    }

    public String getData() {
        return data;
    }

    public void setData(String data) {
        this.data = data;
    }
}

File: src/main/kotlin/com/lykke/matching/engine/daos/Candle.kt
package com.lykke.matching.engine.daos

import com.microsoft.azure.storage.table.TableServiceEntity

class Candle: TableServiceEntity {
    var data: String? = null

    constructor(partitionKey: String, rowKey: String, data: String): super(partitionKey, rowKey) {
        this.data = data
    }
}